doc/nrf/app_dev/device_guides/nrf54l/fota_update.rst

```diff
@@ -0,0 +1,236 @@
+   See :ref:`app_build_output_files` for more information about the contents of update archive.
+
+   .. note::
+      nRF Connect for Desktop does not currently support the FOTA process.
+
+#. Use the `nRF Connect Device Manager`_ mobile app to update your device with the new firmware.
+
+   a. Ensure that you can access the :file:`dfu_application.zip` archive from your phone or tablet.
+   #. In the mobile app, scan and select the device to update.
+   #. Switch to the :guilabel:`Image` tab.
+   #. Tap the :guilabel:`SELECT FILE` button and select the :file:`dfu_application.zip` archive.
+   #. Tap the :guilabel:`START` button.
+
+      .. note::
+         When performing a FOTA update with the iOS app for samples using random HCI identities, ensure that the :guilabel:`Erase application settings` option is deselected before starting the procedure.
+         Otherwise, the new image will boot with random IDs, causing communication issues between the app and the device.
+
+   #. Initiate the DFU process of transferring the image to the device:
+
+      * If you are using an Android device, select a mode in the dialog window, and tap the :guilabel:`START` button.
+      * If you are using an iOS device, tap the selected mode in the pop-up window.
+
+      .. note::
+         For samples using random HCI identities, the Test and Confirm mode should not be used.
+
+   #. Wait for the DFU to finish and then verify that the application works properly.
+
+.. fota_upgrades_over_ble_nrfcdm_common_dfu_steps_end
+
+.. fota_upgrades_outro_end
+
+FOTA update sample
+******************
+
+.. fota_upgrades_update_start
+
+The :zephyr:code-sample:`smp-svr` demonstrates how to set up your project to support FOTA updates.
+
+The sample documentation is from the Zephyr project and is incompatible with the :ref:`ug_multi_image`.
+When working in the |NCS| environment, ignore the part of the sample documentation that describes the building and programming steps.
+In |NCS|, you can build and program the :zephyr:code-sample:`smp-svr` as any other sample using the following commands:
+
+.. tabs::
+
+    .. group-tab:: nRF54l15 SoCs
+
+        .. parsed-literal::
+           :class: highlight
+
+            west build -b *board_target* -- -DEXTRA_CONF_FILE=overlay-bt.conf
+            west flash
+
+.. group-tab:: nRF54l15 SoCs with HW cryptography support
+
+        .. parsed-literal::
+           :class: highlight
+
+            west build -b *board_target* -- -DEXTRA_CONF_FILE=overlay-bt.conf -DSB_CONFIG_BOOT_SIGNATURE_TYPE_ED25519=y -Dmcuboot_CONFIG_NRF_SECURITY=y -Dmcuboot_CONFIG_MBEDTLS=n -Dmcuboot_CONFIG_BOOT_ED25519_PSA=y -Dmcuboot_CONFIG_PM_PARTITION_SIZE_MCUBOOT=0x10000 -Dmcuboot_CONFIG_BOOT_SIGNATURE_USING_KMU=y
+            west flash
+
+
+    .. group-tab:: nRF54l15dk with SPI Flash as update image bank
+
+        .. parsed-literal::
+           :class: highlight
+
+            west build -b *board_target* -T sample.mcumgr.smp_svr.bt.nrf54l15dk.ext_flash
```

Comment:
Since it's only for nRF54L15 maybe there's no point in giving a variable for board target